docs/management/connectors/action-types/cases-webhook.asciidoc

```diff
@@ -0,0 +1,389 @@
+     getIncidentUrl: 'https://testing-jira.atlassian.net/rest/api/2/issue/{{{external.system.id}}}'
+     getIncidentResponseExternalTitleKey: 'key'
+     viewIncidentUrl: 'https://testing-jira.atlassian.net/browse/{{{external.system.title}}}'
+     updateIncidentUrl: 'https://testing-jira.atlassian.net/rest/api/2/issue/{{{external.system.id}}}'
+     updateIncidentMethod: 'put'
+     updateIncidentJson: '{"fields":{"summary":{{{case.title}}},"description":{{{case.description}}},"labels":{{{case.tags}}}'
+     createCommentMethod: 'post',
+     createCommentUrl: 'https://testing-jira.atlassian.net/rest/api/2/issue/{{{external.system.id}}}/comment',
+     createCommentJson: '{"body": {{{case.comment}}}}',
+   secrets:
+     user: testuser
+     password: passwordvalue
+--
+
+`config`:: Defines information for the connector type.
+`hasAuth`::: A boolean that corresponds to *Requires authentication*. If `true`, this connector will require values for `user` and `password` inside the secrets configuration. Defaults to `true`.
+`headers`::: A `record<string, string>` that corresponds to *Headers*.
+`createIncidentUrl`::: A URL string that corresponds to *Create Case URL*.
+`createIncidentMethod`::: A string that corresponds to *Create Case Method*.
+`createIncidentJson`::: A stringified JSON with Mustache variables that corresponds to *Create Case JSON*.
+`createIncidentResponseKey`::: A string from the response body of the create case method that corresponds to the *External Service Id*.
+`getIncidentUrl`::: A URL string with an *External Service Id* Mustache variable that corresponds to *Get Case URL*.
+`getIncidentResponseExternalTitleKey`::: A string from the response body of the get case method that corresponds to the *External Service Title*.
+`viewIncidentUrl`::: A URL string with either the *External Service Id* or *External Service Title* Mustache variable that corresponds to *View Case URL*.
+`updateIncidentUrl`::: A URL string that corresponds to *Update Case URL*.
+`updateIncidentMethod`::: A string that corresponds to *Update Case Method*.
+`updateIncidentJson`::: A stringified JSON with Mustache variables that corresponds to *Update Case JSON*.
+`createCommentUrl`::: A URL string that corresponds to *Create Comment URL*.
+`createCommentMethod`::: A string that corresponds to *Create Comment Method*.
+`createCommentJson`::: A stringified JSON with Mustache variables that corresponds to *Create Comment JSON*.
+
+`secrets`:: Defines sensitive information for the connector type.
+`user`::: A string that corresponds to *User*. Required if `hasAuth` is set to `true`.
+`password`::: A string that corresponds to *Password*. Required if `hasAuth` is set to `true`.
+
+[float]
+[[define-cases-webhook-ui]]
+=== Define connector in {stack-manage-app}
+
+Define Webhook - Case Management connector properties:
+
+[role="screenshot"]
+image::management/connectors/images/cases-webhook-connector.gif[Webhook - Case Management connector]
```

Comment:
> image::management/connectors/images/cases-webhook-connector.gif[Webhook - Case Management connector]

IMO this would be simpler to maintain as a static image, like in the other connector pages. Ideally the UI is intuitive enough that it doesn't require this animation.